CHANGELOG.md

```diff
@@ -1,3 +1,62 @@
+# 0.3.0 (TBD)
+
+## Features
+
+- Upgrade node-mssql to `^4.2.1`. This paves the way for supporting Availability Groups (more testing needed on this front).
+- Better DEBUG logging
+- Added the following properties to the stats output: `config.appName`, `config.encrypt`.
+
+## Bug Fixes
+
+- Added workaround for https://github.com/tediousjs/node-mssql/issues/705 until it's fixed upstream. This isn't a bug in a previous version, but in node-mssql v4.
+
+## Breaking Changes
+
+- Since mssql-pool-party is a wrapper around node-mssql that aims to minimize interface changes, all breaking changes in node-mssql v4 are relevant to this new version, [see here for those changes](https://github.com/tediousjs/node-mssql/blob/7f374a8d73b00b17aa5b5ea5621c4314fc6e2daa/README.md#3x-to-4x-changes).
+- All responses from Request methods (query, execute, etc), for each style (promise, cb, stream), have been unified to match the changes made in v4. They all return a single object (and an error, if applicable). The result object will look something like the following (first example applies to callback/promise/async-await, second example applies to stream). See README.md for richer documentation.
+```
+{
+  output: {},
+  recordset: [{}],
+  recordsets: [
+    [{}],
+  ],
+  returnValue: 0,
+  rowsAffected: [],
+}
+```
+```
+{
+  columns: {
+    ColumnName: {},
+  },
+  output: {},
+  returnValue: 0,
+  rows: [{}],
+  rowsAffected: [],
+}
+```
+- Pool inside `node-mssql`'s ConnectionPool now defaults both `evictionRunIntervalMillis` and `idleTimeoutMillis` to `500ms` (original defaults were `1000ms` and `30000ms` respectively). Can be overridden by modifying `connectionPoolConfig.pool.X`.
```

Comment:
i made that change [because of this](https://github.com/tediousjs/node-mssql/issues/457).  basically, node-mssql wouldn't actually close it's pool connections, which caused jest to hang.  but you have a point, i shouldn't make that the default for everyone, only for the tests.  will fix, thanks for catching that